### ISSUE TITLE
Sign Creation Error

### ISSUE THREAD

**Issue by SteeltonMC** [jakeryang/SimplePaintball#10]
Hello,  MC Paper: #45 Paintball v0.3.4-alpha.3

Sign does not seem to be working. This plugin was working flawlessly a couple Paper Updates ago. I have not touched the plugin or configs- or any other variables. I wanted to move my sign, and this error started coming up. Previously not shown. 

![2020-01-12_11 24 34](https://user-images.githubusercontent.com/45464775/72222091-0f05e100-352f-11ea-8a25-9ca3eeeaac41.png)
[Log.txt](https://github.com/jakeryang/SimplePaintball/files/4050841/Log.txt)

**Comment by jakegut** [comment]
I don't believe I took care of removing signs correctly, which is why the NullPointer shows up: it tries to update a sign that isn't there. I believe commit https://github.com/jakeryang/SimplePaintball/commit/b8d2c460d132391c704f32b278bd5e1c1ab47c42 should take care of it. I will close this when I don't hear about any more sign problems when I release.

**Comment by SteeltonMC** [comment]
This worked, deleting the "sign" information in the Arena's config. 

Needed a full server stop, editing the config to delete the section, save and start.

Editing the config while server is running, and doing a reload; deletes the Arena in question and throws "Internal Server Error"
